Week 4/GroceryStore.html

```diff
@@ -9,8 +9,8 @@
 	<p></p>
 	<script>
 		$("button").click(() => {
-      		const ItemName = $("input#grocery").val();
-      		if (ItemName = "Eggs") {
+      		const itemName = $("input#grocery").val();
```

Comment:
👍 Now I can properly help you with this!

First, make sure you indent everything inside the `{}`.